### ISSUE TITLE
List requirements for pomo

### ISSUE THREAD

**Issue by felixzieger** [mikeevmm/pomo#2]
The necessary requirements for running pomo should be mentioned somewhere in the README or the install script.

For example, on my laptop with os x, I got an error about notify-send:
 ```
 λ ~/projects/pomo/ main pomo
Traceback (most recent call last):
  File "/Users/felix/bin/pomo", line 30, in <module>
    from internal.notify import notify_and_print
  File "/Users/felix/projects/pomo/internal/notify.py", line 7, in <module>
    (run(['notify-send', '-v'], stdout=DEVNULL, stderr=DEVNULL).returncode == 0)
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'notify-send'
 ```

**Comment by mikeevmm** [comment]
Ah right. [`notify-send`][0], by the way, is a part of `libnotify`, usually already included in Linux systems (which is where I tested this).

I'll add a note to the README.

[0]: http://manpages.ubuntu.com/manpages/xenial/en/man1/notify-send.1.html

**Comment by mikeevmm** [comment]
Resolved in d87d43c.
